syntax = "proto2";

package tbclient;

import "tbclient/ActivitySponsor.proto";
import "tbclient/AlaUserInfo.proto";
import "tbclient/AlaLiveInfo.proto";
import "tbclient/Balv.proto";
import "tbclient/ConsumeInfo.proto";
import "tbclient/Ecom.proto";
import "tbclient/EsportInfo.proto";
import "tbclient/GiftInfo.proto";
import "tbclient/GodInfo.proto";
import "tbclient/Icon.proto";
import "tbclient/LikeForumInfo.proto";
import "tbclient/MyGroupInfo.proto";
import "tbclient/NewUser.proto";
import "tbclient/NovelFansInfo.proto";
import "tbclient/NewParrScores.proto";
import "tbclient/PayMemberInfo.proto";
import "tbclient/Pendant.proto";
import "tbclient/PrivSets.proto";
import "tbclient/SimpleUser.proto";
import "tbclient/SpringVirtualUser.proto";
import "tbclient/TbVipInfo.proto";
import "tbclient/ThemeCardInUser.proto";
import "tbclient/TshowInfo.proto";
import "tbclient/TwAnchorProfitItem.proto";
import "tbclient/TwZhiBoUser.proto";
import "tbclient/UserPics.proto";
import "tbclient/UserVideoChannelInfo.proto";
import "tbclient/UserVipInfo.proto";
import "tbclient/VipCloseAd.proto";
import "tbclient/VipShowInfo.proto";

message User {
  optional string BDUSS = 29;
  required ActivitySponsor activity_sponsor = 71;
  optional int32 agree_num = 88;
  required AlaUserInfo ala_info = 78;
  required AlaLiveInfo ala_live_info = 85;
  required Balv balv = 10;
  optional string bawu_type = 26;
  optional string bg_pic = 58;
  optional int32 bimg_end_time = 40;
  optional string bimg_url = 13;
  optional int32 bookmark_count = 53;
  optional int32 bookmark_new_count = 54;
  optional int32 concern_num = 31;
  required ConsumeInfo consume_info = 68;
  optional int32 each_other_friend = 83;
  required Ecom ecom = 74;
  required EsportInfo esport_data = 84;
  optional string fans_nickname = 57;
  optional int32 fans_num = 30;
  optional int64 friend_num = 56;
  optional int32 gender = 42;
  repeated GiftInfo gift_list = 50;
  optional int32 gift_num = 49;
  required GodInfo god_data = 62;
  repeated MyGroupInfo groupList = 48;
  optional int32 has_bottle_enter = 80;
  optional int32 has_concerned = 35;
  optional int32 heavy_user = 63;
  repeated Icon iconinfo = 17;
  optional int64 id = 2;
  optional string intro = 34;
  optional string ios_bimg_format = 22;
  optional string ip = 28;
  optional int32 is_bawu = 25;
  optional int32 is_coreuser = 20;
  optional int32 is_fans = 91;
  optional int32 is_friend = 46;
  optional int32 is_guanfang = 52;
  optional int32 is_huinibuke = 21;
  optional int32 is_interestman = 16;
  optional int32 is_invited = 90;
  optional int32 is_like = 24;
  optional int32 is_login = 1;
  optional int32 is_manager = 11;
  optional int32 is_mask = 43;
  optional int32 is_mem = 39;
  optional int32 is_select_tail = 51;
  optional int32 is_verify = 15;
  optional int32 left_call_num = 89;
  optional int32 level_id = 23;
  repeated LikeForumInfo likeForum = 47;
  optional int32 meizhi_level = 14;
  repeated SimpleUser mute_user = 55;
  optional int32 my_like_num = 33;
  optional string name = 3;
  optional string name_show = 4;
  repeated TshowInfo new_tshow_icon = 65;
  required NewUser new_user_info = 8;
  optional int32 nickname_update_time = 86;
  optional uint32 no_post_high = 73;
  optional int32 no_un = 6;
  required NovelFansInfo novel_fans_info = 60;
  required NewParrScores parr_scores = 59;
  optional string passwd = 36;
  required PayMemberInfo pay_member_info = 41;
  required Pendant pendant = 77;
  optional string portrait = 5;
  optional string portraith = 27;
  optional int32 post_num = 37;
  required PrivSets priv_sets = 45;
  optional int32 priv_thread = 92;
  repeated TwAnchorProfitItem profit_list = 67;
  optional string rank = 12;
  optional string seal_prefix = 79;
  optional int32 sex = 32;
  required SpringVirtualUser spring_virtual_user = 82;
  optional string tb_age = 38;
  required TbVipInfo tb_vip = 72;
  required ThemeCardInUser theme_card = 69;
  optional int32 thread_num = 87;
  optional int32 total_visitor_num = 76;
  repeated TshowInfo tshow_icon = 18;
  required TwZhiBoUser tw_anchor_info = 66;
  optional int32 type = 7;
  repeated UserPics user_pics = 44;
  optional int32 user_type = 19;
  optional int32 userhide = 9;
  required UserVideoChannelInfo video_channel_info = 81;
  required UserVipInfo vipInfo = 61;
  required VipCloseAd vip_close_ad = 70;
  required VipShowInfo vip_show_info = 64;
  optional int32 visitor_num = 75;  
}
